packages/next/src/server/patch-error-inspect.ts

```diff
@@ -0,0 +1,201 @@
+import { findSourceMap } from 'module'
+import type * as util from 'util'
+import { SourceMapConsumer as SyncSourceMapConsumer } from 'next/dist/compiled/source-map'
```

Comment:
This is an older version of `source-map` that's still sync (https://github.com/mozilla/source-map/issues/331). Consider moving to https://www.npmjs.com/package/@cspotcode/source-map-consumer